### DISCUSSION TITLE
关于on_command机器人无响应的问题

### DISCUSSION THREAD

**Discussion by KolinLoa** [nonebot/discussions#388] [问答] [answered]
我机器人的插件基本都是通过logger info进行匹配的，但是商店的插件基本都是用on_command响应的，但是当我导入那些插件时，发信息给机器人，机器人没有回复

**Comment by yanyongyu** [comment, marked as answer]
请提供你的日志，检查你的command start配置

**Reply by yanyongyu** [reply]
请提供你的完整日志以及你的代码
